tests/lib/SimpleSAML/Utils/ConfigTest.php

```diff
@@ -73,8 +74,8 @@ public function testInvalidEnvVariableConfigDirThrowsException()
         $invalidDir = __DIR__.'/e9826ad19cbc4f5bf20c0913ffcd2ce6';
         putenv('SIMPLESAMLPHP_CONFIG_DIR='.$invalidDir);
 
-        $this->expectException(
-            \InvalidArgumentException::class,
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(
             'Config directory specified by environment variable SIMPLESAMLPHP_CONFIG_DIR is not a directory.  '.
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)